plugins/speculation-rules/readme.txt

```diff
@@ -88,6 +88,16 @@ add_filter(
 
 As mentioned above, adding the `no-prerender` CSS class to a link will prevent it from being prerendered (but not prefetched). Additionally, links with `rel=nofollow` will neither be prefetched nor prerendered because some plugins add this to non-idempotent links (e.g. add to cart); such links ideally should rather be buttons which trigger a POST request or at least they should use `wp_nonce_url()`.
 
+= How will this impact analytics and personalization? =
+
+Prerendering can affect analytics and personalization.
+
+For client-side JavaScript, is recommended to delay these until the page clicks and some solutions (like Google Analytics) already do this automatically for prerender. See [Impact on Analytics](https://developer.chrome.com/docs/web-platform/prerender-pages#impact-on-analytics). Additionally, cross-origin iframes are not loaded until activation which can further avoid issues here.
+
+Speculating on hover (moderate) increases the chance the page will be loaded, over preloading without this signal, and thus reduces the risk here. Alternatively, the plugin offers to only speculate on mouse/pointer down (conservative) which further reduces the risk here and is an option for sites which are concerned about this, at the cost of having less of a lead time and so less of a performance gain.
```

Comment:
I find "over preloading without this signal" complicated to understand. What's a "signal"? I definitely think the language here can be simplified.

Is this supposed to mean "speculating on hover increases the chance ... over speculating on eager"?

Also since "speculating on hover (moderate)" is the default in this plugin anyway, I think it's worth formulating this response with that context, just to reiterate here that this is the default that the plugin uses.

Reply:
It's supposed to me "speculating on hover increases the chance of the page actually being visited, and so reduces the chance of any negative impact on analytics due to pages being prerendered (and being included in analytics) but not actually being seen by a user".

I agree the wording could be clearer.

And btw, it may should a bit dismissive to say "hey, if there's a bigger chance of a page being visited, then it's OK if your analytics are off sometimes", but the reality is this already happens in many cases. For example, people opening in a new tab and not actually visiting it. Or open a link, not reading it, but intending to come back to it (but never doing that).